site_root/input/resources/examples/Task-se-001.json

```diff
@@ -0,0 +1,112 @@
+{
+  "resourceType": "Task",
+  "id": "se-001",
```

Comment:
As suggested above, `id` to be `st-001` and the file name to be `Task-st-001.json`?